client/src/containers/guide/PdfViewer.js

```diff
@@ -76,14 +77,14 @@ const PdfViewer = ({ file, defaultPageNumber = 1 }) => {
       {numPages && (
         <>
           <p style={{ textAlign: 'center' }}>
-            Pagina {pageNumber} din {numPages}
+            <Trans>Page</Trans> {pageNumber} <Trans>of</Trans> {numPages}
```

Comment:
Looking [at the docs](https://lingui.js.org/ref/macro.html#trans) it seems that you can leave variables inside `Trans`. This will be helpful if the order of the words is not the same in-between languages.